src/pkg/selectors/sharepoint_test.go

```diff
@@ -0,0 +1,336 @@
+package selectors
```

Comment:
This file was generated by
1. copy-pasting /selectors/onedrive_test.go
2. replacing all "onedrive" -> "sharepoint" and "user" -> "site"

Any code that couldn't compile due to missing dependencies is commented out.